optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/test/java/org/optaplanner/workbench/screens/solver/client/editor/SolverEditorPresenterTest.java

```diff
@@ -125,6 +144,26 @@ public void load() throws Exception {
                                                    path);
     }
 
+    @Test
+    public void testGetContentSupplier() throws Exception {
+
+        final SolverConfigModel content = mock(SolverConfigModel.class);
+
+        doReturn(content).when(presenter).getModel();
```

Comment:
@karreiro Would be better to write
```java
when(presenter.getModel()).thenReturn(content);
```
As Javadoc says:
> Use doReturn() in those rare occasions when you cannot use when(Object).

But you don't have to change it in order to merge this PR.

Reply:
@yurloc fixed.